i18n/en.json

```diff
@@ -1,9 +1,10 @@
 {
 	"@metadata": {
 		"authors": [
-			"Southparkfan"
+			"Southparkfan", "The-Voidwalker"
```

Comment:
Maybe this is personal preference, but so we can keep consistent with our other files, I'd prefer that you add your name on the next line, instead of just after. Just a minor thing.